config/200-controller-clusterrole.yaml

```diff
@@ -100,3 +100,10 @@ rules:
       - "get"
       - "list"
       - "watch"
+  # todo: Do not check-in below lines. This is temporary till eventing controller gets read,list,write access to all channels
```

Comment:
This change is temporary and will not be checked-in.

Reply:
I'll have to check-in this part and revert it after we figure out how to give eventing controller read,list, and write access to all CRD based channels